docs/user-guide/install-zos.md

```diff
@@ -519,93 +537,217 @@ You can obtain the _asid_ from the value of `A=asid` when you issue the followin
 
-2. Run the `zowe-verify.sh` script by issuing the following command:
+You can choose which method to use depending on your familiarity with z/OS configuration steps that are required for the manual path, together with the authority and privileges of your user ID if you choose to run the automated path.
+
+Once the cross memory server is installed and started, there will be started task ZWESIS01 that runs the load library ZWESIS01.  The ZWESIS01 started task serves the ZOWESVR started task and provides secure services that require running in an APF-authorized state.
+
+### Manually installing the Zowe Cross Memory Server
+
+A number of files are included in the USS directory `zowe_install_dir/files/zss`.  If this directory is not present, you must create it by expanding the file `zowe_install_dir/files/zss.pax`.  To do this, first create the folder `zss` beneath `files` using the command `mkdir zss` and navigate into the `zss` folder using the command `cd zss`. Then, expand the `zss.pax` file using the command `pax -ppx -rf ../zss.pax`. 
+
+The manual installation consists of the following steps.
+
+1. ZWESIS01 load module and proclib
+
+    Zowe Cross Memory Server consists of a single load module with the name ZWESIS01.  The load module is supplied in the `files\zss\LOADLIB\ZWESIS01` file.  This must be copied to a user-defined data set `zwes_loadlib`, for example, ZWES.SISLOAD.
+
+    You can copy the ZWESIS01 file to your `zwes_loadlib` data set by using the command `cp ZWESIS01 "//'zwes_loadlib(ZWESIS01)'"`.  The `zwes_loadlib` must be a PDSE due to language requirements.  
+
+    Do not add the `zwes_loadlib` data set to the system LNKLST or LPALST concatenations. You must execute it by using a started task that uses a STEPLIB DD statement so that the appropriate version of the software is loaded correctly.  A sample JCL for the PROCLIB is provided in `files/zss/SAMPLIB/ZWESIS01`.  Copy this to your system PROCLIB, such as SYS1.PROCLIB, or any other PROCLIB in the JES2 Concatenation PROCLIB Path.  
+
+    **Note:**  The user that is assigned to the started task must have an OMVS segment.  The cross memory server loads the module to LPA for its PC-cp services.
+
+2. PPT Entry
+
+    The Zowe cross memory server must run in key 4 and be non-swappable.  For the server to start in this environment, you must add a corresponding PTT entry to the SCHEDxx member of the system PARMLIB. For example, add the following PPT entry to the SCHEDxx member:
 
     ```
-    zowe-verify.sh
+    PPT PGMNAME(ZWESIS01) KEY(4) NOSWAP
     ```
+    After you edit the PARMLIB, issue the following command to make the SCHEDxx changes effective:
 
-The script writes its messages to your terminal window.  The results are marked `OK`, `Info`, `Warning` or `Error`.  Correct any reported errors and restart the Zowe server.  The `zowe-verify.sh` script does not change any settings, so you can run it as often as required.
+    ```
+    /S SET SCH=xx
```

Comment:
Convert to : 
```suggestion
    /S SET SCH=xx
```
/SET SCH=xx

/S SET SCH=xx does not work on my system, /SET SCH=xx does . Any comment from z/OS experts?

Reply:
@Joe-Winchester @dmitrynikolaev @ifakhrutdinov

Reply:
@MarkAckert Nayer is working on a fix of this page: https://github.com/NayerNajafi/docs-site/blob/docs-staging/docs/user-guide/install-zos.md#manually-installing-the-zowe-cross-memory-server

Reply:
This is fixed now in doc. Thanks Nayer and Mark!